### DISCUSSION TITLE
Room 5 - Series 4

### DISCUSSION THREAD

**Discussion by HettiePike** [IBAHCM/RPiR#274] [Archive]
Hettie Pike - HettiePike
Darren Stuart - 2032863s
Neema Zephania - neemazephania

**Comment by HettiePike** [comment]
My repo - https://github.com/IBAHCM/HettiePikeBCI.git

**Comment by neemazephania** [comment]
my repo - https://github.com/IBAHCM/neemazephaniaBCI.git

**Comment by 2032863s** [comment]
my repo - https://github.com/IBAHCM/darrenstuartseries04.git

**Comment by 2032863s** [comment]
Hi guys, installed both of your packages and ran both demos, everything seems to be working fine!

**Reply by neemazephania** [reply]
hi Darren, 
I got an error while installing your package, see attached

<img width="826" alt="Screenshot 2021-02-15 at 12 02 40" src="https://user-images.githubusercontent.com/62024296/107944387-ef3cba00-6f85-11eb-9dad-624fcabe6cdb.png">

I'm wondering if @HettiePike is getting the same or just me?

**Reply by HettiePike** [reply]
I'm having some issues with devtools::install() at the moment so I'm not able to run any packages. But I'll let you know when I can!

**Reply by 2032863s** [reply]
Thank you for highlighting this for me, i forgot to document changes to the namespace. Fixed now though if you would like to try installing again. Cheers

**Reply by HettiePike** [reply]
I'm having this error message:
"Error in find.package(package, lib.loc, verbose = verbose) : 
  there is no package called ‘darrenstuartseries04’"
